### ISSUE TITLE
certbot-dns-route53 plugin performs more validations for the same domain when requesting wildcard + root domain

### ISSUE THREAD

**Issue by vasimi** [certbot/certbot#5928]
`certbot-dns-route53` plugin performs two validations of domain `xxx.xyz` when I request certificate for `*.xxx.xyz` AND `xxx.xyz`. It creates two TXT records and verifies them twice even if it is for the same domain `xxx.xyz`.

**Comment by isayme** [comment]
@MekliCZ just add two seperate txt record.

duplicate with https://github.com/certbot/certbot/issues/5772.

**Comment by MekliCZ** [comment]
Well, this issue isn't about two TXT records (they are handled by the DNS plugin so I don't care about them), but about two validations for the same domain. I don't see any point in validating one domain twice.

**Comment by schoen** [comment]
How do you mean "two validations", @MekliCZ? What behavior do you see that you regard as wrong?

**Comment by dejewi** [comment]
@isayme do you know how to add two same TXT records in AWS Route53?
I get exception: Tried to create resource record set [name='_acme-challenge.example.com.', type='TXT'] but it already exists

**Comment by MekliCZ** [comment]
@schoen I request cert for `*.abc.xyz` and `abc.xyz`. Certbot creates TXT record for `abc.xyz` and verifies `abc.xyz`. Then certbot creates another TXT record for `*.abc.xyz`, but verifies the same domain – `abc.xyz`. So it performs two validations of one domain.

**Comment by richard-fairthorne** [comment]
If you attempt to obtain a certificate like this (root + wildcard) using DNS in manual mode, you are asked to add multiple TXT records, which is impossible with some providers, including GoDaddy.

I have temporarily resorted to getting two seperate certificates.

Would it be possible to verify all the domains using the same TXT record? Perhaps it would address multiple use-cases.

**Comment by dj0x** [comment]
I solved, I discover that have to be created just one record TXT (_acme-challenge.*) with two values (hash given by certbot) separated by line. After run certbot, remember to restart your webserver.

I think depends on your DNS server to setup this. I did it in Route 53 - AWS and this fix this problem.

Obs: Considering wait some seconds (+30) after change your records.

The problem has fixed here:

https://stackoverflow.com/questions/54654135/letsencrypt-certbot-rejects-dns-txt-record-for-wildcard-certificate/55658516#55658516
